app/models/forms/escort.rb

```diff
@@ -0,0 +1,7 @@
+module Forms
+  class Escort < Forms::Base
```

Comment:
This should probably be renamed to match what the form is doing, e.g `CancelEscort`. Seems to be a very specific action to be put into the most generic `Escort` form that normally collects a set of attributes for the entire escort.